conf/quest.config

```diff
@@ -3,26 +3,29 @@
 */
 process {
 
+    //conda = "/projects/b1059/workflows/alignment-nf/env/align.yml"
+    module = "graphviz"
```

Comment:
This will load the graphiz module for every step of the pipeline I believe...

I don't think you need to load it for every step?